.github/workflows/honeypot-dapp.yml

```diff
@@ -18,18 +18,29 @@ on:
         tags:
             - v*
         branches:
-            - develop
-            - adapt-robust-version
+            - add-sepolia-mainnet-builds
 permissions:
     contents: read
     packages: write
     actions: write
 jobs:
-    build:
+    build-localhost:
```

Comment:
It's more beautiful if you use a [matrix](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs)